app/src/main/java/com/amaze/filemanager/ui/fragments/MainFragment.java

```diff
@@ -681,9 +681,12 @@ public boolean onActionItemClicked(ActionMode mode, MenuItem item) {
                     item.getItemId() == R.id.cpy
                         ? PasteHelper.OPERATION_COPY
                         : PasteHelper.OPERATION_CUT;
-
-                PasteHelper pasteHelper = new PasteHelper(getMainActivity(), op, copies);
-                getMainActivity().setPaste(pasteHelper);
+                // Making sure we don't cause an IllegalArgumentException
+                // when passing copies to PasteHelper
+                if(copies.length > 0){
```

Comment:
Why would copies be of length 0? Why is it not null?

Reply:
because it's created like this
```java
HybridFileParecelable[] copies = new HybridFileParecelable[checkedItems.size()];
```

Reply:
Still why would copies be of length 0?

Reply:
I also have the same question, I tried to replicate the bug but I couldn't

Reply:
Try to use the code paths to find what causes the issue.

Reply:
Can you explain more?

Reply:
Without replicating, you can analyze the code and the error to find where it could have happened and why.

Reply:
After looking into the crash log I've found that the error is triggered by `onActionItemClicked()` method in `MainFragment`, the crash is a bit weird because it seems the app logic hides the action bar when there is no file selected, so there is no way `checkedItems` will be empty, but I added a check condition anyway to make sure it does not happen again.